traffic_ops/traffic_ops_golang/invalidationjobs/invalidationjobs.go

```diff
@@ -130,11 +156,35 @@ WHERE server.status NOT IN (
 const updateQuery = `
 UPDATE job
 SET asset_url=$1,
-    keyword=$2,
-    parameters=$3,
-    start_time=$4
+    ttl_hr=$2,
+    start_time=$3
+WHERE job.id=$4
+RETURNING asset_url,
+          (
+           SELECT tm_user.username
+           FROM tm_user
+           WHERE tm_user.id=job.job_user
+          ) AS created_by,
+          (
+           SELECT deliveryservice.xml_id
+           FROM deliveryservice
+           WHERE deliveryservice.id=job.job_deliveryservice
+          ) AS delivery_service,
+          job.id,
+          'PURGE' as keyword,
+          CONCAT('TTL:', ttl_hr, 'h') AS parameters,
+          start_time
+`
+
+// Almost the same as updateQuery, but returns appropriate values for API 4.0+
+const updateQueryV40 = `
```

Comment:
We should probably call this V4 instead of V40, because we'll likely just update this query with new fields if they're ever added.